### PR TITLE
Tests for extenstions

### PR DESCRIPTION
**Pull request by tadhg-ohiggins**
Add a shell for extension tests, a minimal test for the US Code preprocessor, and rename ``extend_regparser`` to ``atf_regparser``.

### CODE REVIEW

File: eregs_extensions/atf_regparser/tests/preprocessor_tests.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# vim: set encoding=utf-8
+from unittest import TestCase
+
+from tests.xml_builder import XMLBuilderMixin
```
Comment (cmc333333) [inline eregs_extensions/atf_regparser/tests/preprocessor_tests.py]: Bleh. I suppose this means we should move the `tests` module from `regparser` into `regparser.tests`?

**Comment by cmc333333** [comment]
This is a good example we can build from

**Comment by cmc333333** [comment]
@tadhg-ohiggins does the requirements file need to be updated, too? I wonder why travis didn't run...

File: eregs_extensions/atf_regparser/tests/preprocessor_tests.py (hunk 1 above)
Comment (tadhg-ohiggins) [inline eregs_extensions/atf_regparser/tests/preprocessor_tests.py]: Yes, I think we have to do that, otherwise namespace collisions seem too likely…

**Pull request merged**